config/packages/guides.yml

```diff
@@ -0,0 +1,14 @@
+guides:
+  theme: doctrine
+  links_are_relative: true
+  themes:
+    doctrine: '%doctrine.website.root_dir%/templates/guides'
+
+code:
+  aliases:
+      html+php: php
+      html+jinja: html
+      php-annotations: php
+      php-attributes: php
+  languages:
```

Comment:
What does this do?

Reply:
good catch, I removed part of the config, this config is used by the guides di extensions for symfony DI, to configure the library and all services.